src/main/java/subway/section/Section.java

```diff
@@ -0,0 +1,57 @@
+package subway.section;
+
+public class Section {
+
+    public static final Long WRONG_ID = -1L;
+    public static final Section DO_NOT_EXIST_SECTION = null;
+
+    private Long id;
+    private Long lineId;
+    private Long stationId;
+    private int distance;
+    private Long nextStationId;
+
+    public Section(Long id, Long lineId, Long stationId, int distance, Long nextStationId) {
+        this.id = id;
+        this.lineId = lineId;
+        this.stationId = stationId;
+        this.distance = distance;
+        this.nextStationId = nextStationId;
+    }
+
+    public Section(Long lineId, Long stationId, int distance, Long nextStationId) {
+        this.lineId = lineId;
+        this.stationId = stationId;
+        this.distance = distance;
+        this.nextStationId = nextStationId;
```

Comment:
```suggestion
        this(null, lineId, stationId, distance, nextStationId);
```
매번 새로 쓰기보단, 기존 생성자를 활용하는 방법도 있어요. 중복을 제거하는데 도움을 줍니다 😄

Reply:
꿀팁 감사합니다! 생성자 작성을 최대한 하나만 하도록 변경하였습니다.